doc/org-roam.org

```diff
@@ -652,28 +652,9 @@ Since version 9.5, Org has first-class support for citations. Org-roam supports
 the caching of both these in-built citations (of form ~[cite:@key]~) and [[https://github.com/jkitchin/org-ref][org-ref]]
 citations (of form cite:key).
 
-** Setup
-
-*org-ref*
-
-To ensure that org-ref citations are cached, ~org-ref~ must be loaded prior to
-calls of ~org-roam-db-sync~. This can be achieved by placing ~(require
-'org-ref)~ before ~org-roam~ is loaded, for example in the ~:init~ block of the
-Org-roam use-package declaration:
-
-#+begin_src emacs-lisp
-(use-package org-roam
-  :init
-  (require 'org-ref)
-  :config
-  ;;...
-  )
-#+end_src
-
-*org-cite*
-
-Org-cite should be loaded by default, and no additional setup should be
-required.
+Org-roam attempts to load both the ~org-ref~ and ~org-cite~ package when
```

Comment:
What does this mean? How will you know "when needed"?

That if you have them installed, it will load them, but otherwise do nothing?

I was wondering about a user who has been using org-ref, but is migrating to org-ref-cite. For that user, both will be loaded until they uninstall org-ref?

Reply:
@bdarcus It means that each time when org-roam will need to parse files it will try to automatically load both of the packages. If it detects that the package is loaded, it will parse cite links from the available packages. Otherwise, it will simply noop their parsing and do nothing. 

> I was wondering about a user who has been using org-ref, but is migrating to org-ref-cite. For that user, both will be loaded until they uninstall org-ref?

Yes. If org-roam detects that both, org-cite` and `org-ref are available, both of the citations types will be parsed and stored in the database.